src/layman/layer/db/db_test.py

```diff
@@ -298,6 +304,16 @@ def test_guess_scale_denominator(country110m_table, country50m_table, country10m
     assert sd_5k == 5000
 
 
+@pytest.mark.parametrize("posted_layer", [
+    pytest.param(('25k_vertexes', 'sample/layman.layer/25k_vertexes.geojson'), id='25k_vertexes',
```

Comment:
I can not see the 'sample/layman.layer/25k_vertexes.geojson' file being committed anywhere.